third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -566,16 +598,21 @@ function Pause_evaluateInFrame({ frameId, expression }) {
         callFrameId: frame.callFrameId,
         expression,
         objectGroup: REPLAY_CDT_PAUSE_OBJECT_GROUP
-      }
+      },
+      false
```

Comment:
as a general principle when calling methods with boolean params like this it would be good to have a comment like `/* throwOnApiError */ false` instead of `false` to improve readability (an enum will enhance readability even more though that's often overkill).